cli/arguments/fqbn.go

```diff
@@ -69,17 +69,15 @@ func (f *Fqbn) Set(fqbn string) {
 // - it tries to autodetect the board connected to the given port flags
```

Comment:
Please update the reference to `sketch.json` in the comment at line 68 (line is out of the range allowed by PR review suggestions system):

https://github.com/arduino/arduino-cli/blob/ca3929706b537a871dd3804421161a0175dac921/cli/arguments/fqbn.go#L68

Reply:
There was also another instance of `sketch.json` in the sketch specification: https://github.com/arduino/arduino-cli/pull/1930/commits/eecd894e635d8c59cce228b5f0dcacbf5d637a7e